packages/app/src/components/time-series-chart/components/tooltip/tooltip-series-list.tsx

```diff
@@ -62,6 +62,8 @@ export function TooltipSeriesList<T extends TimestampedValue>({
   ) {
     const numberValue = (value[key] as unknown) as number | null;
 
+    if (numberValue === null) return '-';
```

Comment:
```suggestion
    if (numberValue === null) return '-';

    return isPresent(numberValue)
      ? isPercentage
        ? `${formatPercentage(numberValue)}%`
        : formatNumber(numberValue)
      : '-';
  }```

Reply:
isPresent is already catching null values. So I think we only need this part. Otherwise the second isPresent is only catching undefined and the checks are partially redundant.

```suggestion
    return isPresent(numberValue)
      ? isPercentage
        ? `${formatPercentage(numberValue)}%`
        : formatNumber(numberValue)
      : '-';
```